README.md

```diff
@@ -200,6 +200,10 @@ If you use the `bundled` features, you will get pregenerated bindings for the
 bundled version of DuckDB. If you want to run `bindgen` at buildtime to
 produce your own bindings, use the `buildtime_bindgen` Cargo feature.
 
+## Rust version compatibility
+
+duckdb-rs is built and tested with stable Rust, and will keep a rolling MSRV (minimum supported Rust version) that can only be updated in major releases on a need by basis (e.g. project dependencies bump their MSRV or a particular Rust feature is useful for us etc.). The new MSRV will be at least 6 months old. Minor releases are guaranteed to have the same MSRV.
```

Comment:
@Maxxen What do you think?

Reply:
I'm not sure if "Minor releases are guaranteed to have the same MSRV" makes sense for us, given the infrequent major releases. For example, if we need a new crate or update an existing one that requires a higher MSRV, we wouldn't be able to update before the next major DuckDB release.

Reply:
I changed the last sentence to say "Patch releases" for it to make sense for our use case.